assets/css/_search_form.scss

```diff
@@ -105,3 +115,40 @@
   }
 }
 // #endregion Search Input Buttons
+
+// #region Autocomplete Control
+// Hide the autocomplete control by default
+.c-search-form__autocomplete-container {
+  display: none;
+}
+
+// When input controls have focus --
+.c-search-form__search-control[data-autocomplete-visible="true"]:focus-within {
+  // -- Show Autocomplete control
+  .c-search-form__autocomplete-container {
+    display: block;
+  }
+
+  // -- Round out the bottom of the input bar
```

Comment:
**most nitpick-y of nitpicks (non-blocking):** isn't this actually squaring off, rather than rounding out, the bottom of the input bar?

Reply:
🤣, thank you for catching this!!!
73bdffb34b3304656a02595cafd307433828c362

Reply:
another nit: should we move the border radius styling of the autocomplete into this file and make `.c-autocomplete`'s `border-radius` `4px` all around?